charts/nextcloud/templates/cronjob.yaml

```diff
@@ -51,9 +51,9 @@ spec:
                 - "--fail"
                 - "-L"
               {{- if .Values.ingress.tls }}
-                - "https://{{ .Values.nextcloud.host }}/cron.php"
+                - "https://{{ .Values.nextcloud.host }}{{ .Values.ingress.path }}/cron.php"
               {{- else }}
-                - "http://{{ .Values.nextcloud.host }}/cron.php"
+                - "http://{{ .Values.nextcloud.host }}{{ .Values.ingress.path }}/cron.php"
```

Comment:
With the current default it renders to `http://host//cron.php` is that intended?